src/components/Layout/Header/DegradedStateBanner.tsx

```diff
@@ -34,18 +34,8 @@ export const DegradedStateBanner = memo(() => {
   const translate = useTranslate()
   const footerBg = useColorModeValue('blackAlpha.100', 'whiteAlpha.100')
   const buttonBg = useColorModeValue('blackAlpha.100', 'whiteAlpha.100')
-  const portfolioLoadingStatusGranular = useSelector(selectPortfolioLoadingStatusGranular)
   const assets = useSelector(selectAssets)
-
-  const erroredAccountIds = useMemo(() => {
-    return entries(portfolioLoadingStatusGranular).reduce<AccountId[]>(
-      (acc, [accountId, accountState]) => {
-        accountState === 'error' && acc.push(accountId)
-        return acc
-      },
-      [],
-    )
-  }, [portfolioLoadingStatusGranular])
+  const erroredAccountIds = useSelector(selectPortfolioErroredAccountIds)
```

Comment:
was originally going to use this logic to drive the balance history errors before I realized it needs to be tracked separately, but may as well keep this in a selector now its there